tests/test_subscription.py

```diff
@@ -248,6 +248,15 @@ def test_create_and_update_and_list_subscription(rse_factory, rest_client, auth_
     assert response.status_code == 200
     assert loads(loads(response.get_data(as_text=True))['filter'])['project'][0] == 'toto'
 
+    # Set the subscription as inactive
+    data = {'options': {'state': 'I'}}
+    response = rest_client.put('/subscriptions/root/' + subscription_name, headers=headers(auth(auth_token)), json=data)
+    assert response.status_code == 201
+
+    response = rest_client.get('/subscriptions/root/' + subscription_name, headers=headers(auth(auth_token)))
+    assert response.status_code == 200
+    assert loads(response.get_data(as_text=True))['state'] == 'INACTIVE'
```

Comment:
We should add an earlier assertion that ensures that the state was not `INACTIVE` before, to actually verify that this update action is changing the state.